src/frontend/planner_test/tests/testdata/output/temporal_filter.yaml

```
@@ -36,18 +36,121 @@
         └─StreamExchange { dist: Broadcast }
           └─StreamProject { exprs: [SubtractWithTimeZone(now, '00:15:00':Interval, 'UTC':Varchar) as $expr2], output_watermarks: [$expr2] }
             └─StreamNow { output: [now] }
-- name: Temporal filter fails without `now()` in lower bound
+- name: Temporal filter with `now()` in upper bound
   sql: |-
     create table t1 (ts timestamp with time zone);
     select * from t1 where now() - interval '15 minutes' > ts;
-  stream_error: All `now()` exprs were valid, but the condition must have at least one now expr as a lower bound.
+  stream_plan: |-
+    StreamMaterialize { columns: [ts, t1._row_id(hidden)], stream_key: [t1._row_id], pk_columns: [t1._row_id], pk_conflict: NoCheck }
+    └─StreamDynamicFilter [condition_always_relax] { predicate: (t1.ts < $expr1), output: [t1.ts, t1._row_id] }
```

Why not following the original style?

```suggestion
    └─StreamDynamicFilter { predicate: (t1.ts < $expr1), output: [t1.ts, t1._row_id], condition_always_relax: true }
```

---

yes, I will fix it.. but it make me think about the border of the style for we have has `StreamHashAgg [append_only, eowc]`